.github/workflows/trivy-security-scan.yml

```diff
@@ -39,47 +39,93 @@ jobs:
           echo "::set-output name=available::true"
         shell: bash
 
-  trivy_scan_image:
+  update-trivy-db:
     needs: wait-for-image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup oras
+        uses: oras-project/setup-oras@9c92598691bfef1424de2f8fae81941568f5889c # v1.21
+
+      - name: Get current date
+        id: date
+        run: echo "date=$(date +'%Y-%m-%d')" >> $GITHUB_OUTPUT
+
+      - name: Download and extract the vulnerability DB
+        run: |
+          mkdir -p $GITHUB_WORKSPACE/.cache/trivy/db
```

Comment:
`$GITHUB_WORKSPACE` is already the default, why using it explicitly?

Reply:
Just my preference for readability but I have changed it.